snippets/logging-clusterlogging-cr-snip.adoc

```diff
@@ -0,0 +1,23 @@
+// Text snippet included in the following assemblies:
+// logging/logging-loki-ocp.adoc
+//
+// Text snippet included in the following modules:
+//
+//
+:_content-type: SNIPPET
+
+[source,yaml]
+----
+apiVersion: logging.openshift.io/v1
+kind: ClusterLogging
+metadata:
+  name: instance
+  namespace: openshift-logging
+spec:
+  logStore:
+    type: lokistack
+    lokistack:
+      name: lokistack-dev
```

Comment:
I believe storage team would prefer to call this something other then 'lokistack-dev'

Reply:
Yep, that should be `logging-loki`